sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseGridDetailed.java

```diff
@@ -1,25 +1,31 @@
 package de.symeda.sormas.ui.caze;
 
+import java.util.Date;
 import java.util.List;
 import java.util.stream.Stream;
 
+import com.vaadin.ui.renderers.DateRenderer;
 import com.vaadin.ui.renderers.TextRenderer;
 
 import de.symeda.sormas.api.FacadeProvider;
 import de.symeda.sormas.api.caze.AgeAndBirthDateDto;
 import de.symeda.sormas.api.caze.CaseCriteria;
 import de.symeda.sormas.api.caze.CaseIndexDetailedDto;
+import de.symeda.sormas.api.i18n.Captions;
 import de.symeda.sormas.api.i18n.I18nProperties;
 import de.symeda.sormas.api.person.PersonHelper;
 import de.symeda.sormas.api.utils.SortProperty;
 import de.symeda.sormas.ui.ControllerProvider;
+import de.symeda.sormas.ui.utils.DateFormatHelper;
 import de.symeda.sormas.ui.utils.ShowDetailsListener;
 import de.symeda.sormas.ui.utils.UuidRenderer;
 
 public class CaseGridDetailed extends AbstractCaseGrid<CaseIndexDetailedDto> {
 
 	private static final long serialVersionUID = 3734206041728541742L;
 
+	private static final String LATEST_SAMPLE_DATE_TIME_AND_SAMPLE_COUNT = Captions.CaseData_latestSampleDateTime;
```

Comment:
Use `I18nProperties.getPrefixCaption`

Reply:
I did it like this because it's been done the same in other grid classes, and it works.
I'm not reluctant to change this at all, but I kinda wanna understand why would it be better to use a random string here and add a .setCaption(I18nProperties.getPrefixCaption(blabla, blabla)) below ?
For the sake of clarity, to remove a little bit of "magic" ?

Reply:
Mainly because it's more consistent. The Captions file is auto-generated, so it can easier break things if the property is renamed. The properties used for getPrefixCaption are manually added by the developers. It's true that we're using Captions.xxx in other grids, but I'd say that's probably the less ideal way.